RUFAS/routines/animal/animal_manager.py

```diff
@@ -777,12 +777,61 @@ def _create_default_pen(
             max_stocking_density=max_stocking_density,
         )
 
+    @classmethod
+    def _create_duplicate_pen(
+        cls,
+        pen_id: int,
+        animal_combination: AnimalCombination,
+        num_stalls: int,
+        max_stocking_density: float,
+        reference_pen: Pen,
+    ) -> Pen:
+        """
+        Create a default Pen object with the given parameters.
+
+        Parameters
+        ----------
+        pen_id : int
+            The unique identifier for the pen.
+        animal_combination : AnimalCombination
+            The animal combination for the pen.
+        num_stalls : int
+            The number of stalls in the pen.
+        max_stocking_density : float
+            The maximum stocking density for the pen.
+        reference_pen : Pen
+            Pen object that has more animals than available space.
+
+        Returns
+        -------
+        Pen
+            A new Pen object with the specified parameters and duplicate values for other attributes of reference pen.
+
+        """
+
+        return Pen(
+            pen_id=pen_id,
+            pen_name=str(pen_id),
+            vertical_dist_to_milking_parlor=reference_pen.vertical_dist_to_parlor,
+            horizontal_dist_to_milking_parlor=reference_pen.horizontal_dist_to_parlor,
+            number_of_stalls=num_stalls,
+            housing_type=reference_pen.housing_type,
+            bedding_type=reference_pen.bedding_type,
+            pen_type=reference_pen.pen_type,
+            manure_handling=reference_pen.manure_handling,
+            manure_separator=reference_pen.manure_separator,
+            manure_separator_after_digestion=reference_pen.manure_separator_after_digestion,
+            manure_storage=reference_pen.manure_storage,
+            animal_combination=animal_combination,
+            max_stocking_density=max_stocking_density,
+        )
+
     def _create_default_pens_for_potential_space_shortage(
```

Comment:
```suggestion
    def _create_additional_pens_for_potential_space_shortage(
```
I think this name would better reflect this method's purpose.